Source/WebCore/dom/Document.h

```diff
@@ -2570,6 +2573,7 @@ class Document
 #endif
 
     bool m_hasViewTransitionPseudoElementTree { false };
+    bool m_renderingSuppressedForViewTransition { false };
```

Comment:
`m_renderingIsSuppressedForViewTransition`

Reply:
The consequence of this bit getting left set by accident are catastrophic (tabs that stop rendering). I think we need some release logging or protection against botched view transitions leaving things in a bad state.